lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -234,8 +239,11 @@ def add_token(xml, token)
         xml.ssl_token token
```

Comment:
We probably need to update this method to either pass down options to grab token from there if payment is a credit card but has options[:ssl_token] or to return unless token.is_a?(String)

Reply:
Good catch... I thought I'd added enough test variations to cover that situation 😓

Oh wait, I think it didn't catch them because we have another place where we're adding `ssl_token` in `add_auth_purchase_params` 😅